tests/test_repository.py

```diff
@@ -40,6 +56,8 @@ def test_content_structure_is_preserved():
 
 
 def test_iterables_are_flattened():
+    """Test that iterables structures are changed and flattened"""
+
```

Comment:
You've added docstrings with whitespace following them and added docstrings elsehwere without a newline. Let's just remove the new lines afterwards since that's consistent with the style everywhere else in this project

Reply:
Okay so something like

```
def test_iterables_are_flattened():
    """Test that iterables structures are changed and flattened"""
     data = {
        "platform": ["UNKNOWN"],
    }
....
```

Reply:
Yes

Reply:
Docstring newlines removed in latest commit